### DISCUSSION TITLE
[Bug Report]Vant4移动端组件库 Pagination 组件在 uniapp 中样式丢失

### DISCUSSION THREAD

**Discussion by Orange-520** [youzan/vant#12234] [Q&A]
### 重现链接

https://github.com/youzan/vant

### Vant 版本

4.6.6

### 描述一下你遇到的问题。

Pagination组件样式丢失

### 重现步骤

1、npm i vant
2、main.js 中全局引入，全局注册：
import Vant from 'vant'
import 'vant/lib/index.css'
app.use(Vant)

3、正常使用
<van-pagination force-ellipses v-model="currentPage" :total-items="total" :items-per-page="pageSize"></van-pagination>

4、样式丢失

### 设备/浏览器

Edge

**Comment by inottn** [comment]
https://vant-ui.github.io/vant/#/zh-CN/faq#shi-fou-zhi-chi-zai-uni-app-zhong-shi-yong

**Reply by Orange-520** [reply]
谢谢你，那我得搜索一下uniapp vue3 应该使用什么组件库了
